udata_front/theme/gouvfr/assets/js/components/search/OrganizationSearch/OrganizationSearch.vue

```diff
@@ -36,15 +37,16 @@
           :id="option.id"
           role="option"
           :aria-selected="isSelected(option.id)"
+          @mousedown.prevent="preventCollapse"
+          @click="openLink(option.link, $event)"
```

Comment:
We shouldn't have a `@click` on a `<li>` to open a link. `OrganizationSearchOption` should use a `<a>` instead.

This is already done in `MenuSearchOption` but should be updated too to have the `<a>` around the text and a `fr-enlarge-link` on the parent to extend it to the option.

Existing examples doing this can be found in [reuse component](https://github.com/datagouv/udata-front/blob/master/udata_front/theme/gouvfr/assets/js/components/Reuse/Reuse.vue#L2) and [dataset component](https://github.com/datagouv/udata-front/blob/master/udata_front/theme/gouvfr/datagouv-components/src/components/DatasetCard/DatasetCard.vue).